### ISSUE TITLE
Missing NSLocationAlwaysUsageDescription in Info.plist without any GeoLocation Usage in 0.61.5 (ITMS-90683)

### ISSUE THREAD

**Issue by felix-voicehub** [facebook/react-native#28200]
Description:

I am getting E-Mail from Apple after submitting to App Store Connect regarding ITMS-90683. It basically says that I have to include `NSLocationAlwaysUsageDescription` and `NSLocationWhenInUseUsageDescription` in my Info.plist because they say that my

> app's code references one or more APIs that access sensitive user data.

Now, I do not use any of react natives geolocation functions. I have scanned my whole repo for anything regarding these APIs, and the only thing which came up was `RCTConvert+CoreLocation.h` (https://github.com/facebook/react-native/blob/master/React/Views/RCTConvert%2BCoreLocation.h) and it's ".m" file. These do import "CoreLocation", which could be enough to trigger this E-Mail, as suggested [here](https://github.com/facebook/react-native/issues/20879#issuecomment-592505530).

See this issue for explaining the whole background: https://github.com/facebook/react-native/issues/20879.

React Native version:
```
System:
    OS: macOS Mojave 10.14.6
    CPU: (8) x64 Intel(R) Core(TM) i7-4980HQ CPU @ 2.80GHz
    Memory: 22.33 MB / 16.00 GB
    Shell: 5.3 - /bin/zsh
  Binaries:
    Node: 12.12.0 - /usr/local/bin/node
    npm: 6.13.6 - /usr/local/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  SDKs:
    iOS SDK:
      Platforms: iOS 13.2, DriverKit 19.0, macOS 10.15, tvOS 13.2, watchOS 6.1
    Android SDK:
      API Levels: 28, 29
      Build Tools: 28.0.3, 29.0.2
      System Images: android-28 | Intel x86 Atom_64, android-29 | Google APIs Intel x86 Atom
  IDEs:
    Android Studio: 3.5 AI-191.8026.42.35.5977832
    Xcode: 11.3.1/11C504 - /usr/bin/xcodebuild
  npmPackages:
    react: 16.9.0 => 16.9.0 
    react-native: 0.61.5 => 0.61.5 
  npmGlobalPackages:
    react-native-eject: 0.1.2
```

## Steps To Reproduce

1. Bootstrap a clean react native app.
2. Submit it to App Store connect.

## Expected Results

As Geolocation was supposed to be extracted from the core, I wonder why I still get this warning. I do not consider adding the mentioned values in my plist, as I expect that it will most likely show on some user facing screen that the app is using location services when it is definitely not.

**Comment by hramos** [comment]
Thank you for filing a new issue. The old one was closed after Geolocation was removed from core, but it looks like this needs some follow up.

**Comment by maschad** [comment]
The `<CoreLocation/CoreLocation.h>` is being imported in the `RCTCameraRollManager.mm`  [class](https://github.com/facebook/react-native/blob/6f54f00e0d3f66d338068a7d2a5f192e87dcc0e7/Libraries/CameraRoll/RCTCameraRollManager.mm#L11), which is actually scheduled to be removed from Core as mentioned [here](https://github.com/facebook/react-native/issues/23313#issue-407288960)  - I am not too sure why **CameraRollManager** would require it though as it is unused in that class.

**Comment by felix-voicehub** [comment]
@maschad Thanks for the PR. Hope this will get merged.

**Comment by maschad** [comment]
Did apple approve your build after you applied the fix? @felix-voicehub

**Comment by felix-voicehub** [comment]
@maschad I'm only on testflight currently. With "fix" you mean just manually applying your PR to my repo?

**Comment by maschad** [comment]
@felix-voicehub yes, or you could do it via `npm` using the command ```npm install --save maschad/react-native/tree/remove-corelocation```

**Comment by felix-voicehub** [comment]
@maschad It took me a while but I finally applied the 2 PRs (https://github.com/facebook/react-native/pull/28326/files and https://github.com/facebook/react-native/pull/28300/files) to my own repo.

I am still getting an E-Mail from Apple, however, instead of 2 notices (`NSLocationAlwaysUsageDescription` and `NSLocationWhenInUseUsageDescription`), there is now only 1 item with `NSLocationWhenInUseUsageDescription` in that e-mail. I have no idea where that comes from.

**Comment by fritzfr** [comment]
Yes this is still an issue. Just tried again with a new RN project on 0.63.1.

**Comment by cwilby** [comment]
Super sketch.

**Comment by Muha-coinmotion** [comment]
Yes this is still an issue. Just tried again with a new RN project on 0.72
